templates/release.yml

```diff
@@ -0,0 +1,60 @@
+name: Publish Module
+
+on: workflow_dispatch
+  
+jobs:
+  publish-module:
+    name: Tag Release and Publish to Forge
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+        with:
+          ref: ${{ github.ref }}
+          clean: true
+          fetch-depth: 0
+      - name: Get Version
+        id: gv
+        run: |
+          echo "::set-output name=ver::$(jq --raw-output .version metadata.json)"
+      - name: Install the PDK
+        run: |
+          source /etc/os-release
+          wget https://apt.puppet.com/puppet-tools-release-${UBUNTU_CODENAME}.deb
+          sudo dpkg -i puppet-tools-release-${UBUNTU_CODENAME}.deb
```

Comment:
Would you make this same change to the other gh action too, please?

Reply:
Yes